inventory/admin.py

```diff
@@ -27,6 +30,41 @@ class OwnershipAdmin(admin.ModelAdmin):
 admin.site.register(Ownership, OwnershipAdmin)
 
 
+class OwnershipValueProxy(Ownership):
```

Comment:
Migrations aren't necessary as this function only as a stand-in shell and should and does not do anything on the database

Reply:
The main problem I have is that migrations will be created for it anyway if, at some point in time, we run `python manage.py makemigrations`, even if that's done in a completely unrelated feature branch. It's possible to only run `makemigrations` for a specific app to avoid this, but that sounds like a workaround that someone is going to forget to do at some point.